stl/inc/ranges

```diff
@@ -34,7 +34,10 @@ namespace ranges {
     template <class _Rng>
     concept viewable_range = range<_Rng>
         && ((view<remove_cvref_t<_Rng>> && constructible_from<remove_cvref_t<_Rng>, _Rng>)
-            || (!view<remove_cvref_t<_Rng>> && borrowed_range<_Rng>));
+            || (!view<remove_cvref_t<_Rng>>
+                && (is_lvalue_reference_v<_Rng>
+                    || (movable<remove_reference_t<_Rng>>
+                        && !_Is_specialization_v<remove_cvref_t<_Rng>, initializer_list>))));

```

Comment:
The gift that never stops giving

Reply:
Can we reuse `_Is_initializer_list` here too?

Reply:
Good idea - I'll have to bring back the `remove_cvref_t`, which will again justify it being a distinct trait.